RELEASE/scripts/autoscend.ash

```diff
@@ -2492,86 +2622,91 @@ boolean LX_attemptPowerLevel()
 		}
 	}
 
+	//scaling damage zones
 	if(elementalPlanes_access($element[stench]) && auto_have_skill($skill[Summon Smithsness]) && (get_property("auto_beatenUpCount").to_int() == 0))
 	{
-		autoAdv(1, $location[Uncle Gator\'s Country Fun-Time Liquid Waste Sluice]);
+		if(autoAdv($location[Uncle Gator\'s Country Fun-Time Liquid Waste Sluice])) return true;
 	}
-	else if(elementalPlanes_access($element[spooky]))
+	if(elementalPlanes_access($element[spooky]))
 	{
-		autoAdv(1, $location[The Deep Dark Jungle]);
+		if(autoAdv($location[The Deep Dark Jungle])) return true;
 	}
-	else if(elementalPlanes_access($element[cold]))
+	if(elementalPlanes_access($element[cold]))
 	{
-		autoAdv(1, $location[VYKEA]);
+		if(autoAdv($location[VYKEA])) return true;
 	}
-	else if(elementalPlanes_access($element[sleaze]))
+	if(elementalPlanes_access($element[sleaze]))
 	{
-		autoAdv(1, $location[Sloppy Seconds Diner]);
+		if(autoAdv($location[Sloppy Seconds Diner])) return true;
 	}
-	else if (elementalPlanes_access($element[hot]))
+	if (elementalPlanes_access($element[hot]))
 	{
-		autoAdv(1, $location[The SMOOCH Army HQ]);
+		if(autoAdv($location[The SMOOCH Army HQ])) return true;
 	}
-	else if (neverendingPartyAvailable())
+	if (neverendingPartyAvailable())
 	{
-		neverendingPartyPowerlevel();
+		if(neverendingPartyPowerlevel()) return true;
 	}
-	else
+	if(unlockThinknerdWarehouse())
 	{
-		// burn all spare clovers after level 12 if we need to powerlevel.
-		int cloverLimit = get_property("auto_wandOfNagamar").to_boolean() ? 1 : 0;
-		if (my_level() >= 12 && internalQuestStatus("questL12War") > 1 && cloversAvailable() > cloverLimit)
-		{
-			//Determine where to go for clover stats, do not worry about clover failures
-			location whereTo = $location[none];
-			switch (my_primestat())
-			{
-				case $stat[Muscle]:
-					whereTo = $location[The Haunted Gallery];
-					break;
-				case $stat[Mysticality]:
-					whereTo = $location[The Haunted Bathroom];
-					break;
-				case $stat[Moxie]:
-					whereTo = $location[The Haunted Ballroom];
-					break;
-			}
+		if(autoAdv($location[The Thinknerd Warehouse])) return true;
+	}
+	//else if(
+	//TODO add a different function that spends a wish or adventures trying to unlock thinknerdWarehouse.
 
-			cloverUsageInit();
-			boolean retval = autoAdv(1, whereTo);
-			cloverUsageFinish();
-			return retval;
+	// use spare clovers to powerlevel
+	int cloverLimit = get_property("auto_wandOfNagamar").to_boolean() ? 1 : 0;
+	if(cloversAvailable() > cloverLimit)
```

Comment:
This will spam all our clovers regardless of what level we are. I suggest you put the check back for level 12 at the very least if not revert this line completely.
Checking the war is complete was a bit of hack by me (should really check we have no outstanding quests that would benefit from clovers which is a bigger refactor) but the level 12 check most definitely should remain.

Reply:
Isn't spamming our clovers here a good thing? each clover spent here saves many turns of trying to find the NC adventure in that zone, saving adv is the goal of a clover in the first place. Probably 8 or so adv saved per clover if you are in any older path... maybe a noncombat check instead? since the amount of adv saved directly depends on how much noncombat you can get?

In testing I had it genuinely hit this when i needed to power levlevel from level 7 to level 8, it used a single clover and a single adv here, successfully leveled up to level 8 from that one single clover, and then moved on to do quests. Thus drastically cutting down on the amount of adventures we spent here.

Reply:
If you want to do all the refactoring to figure out how many turns each clover used saves then be my guest but until then, lets not waste all the users clovers until we know they don't need them.
Otherwise you can have the pleasure of answering all of the "why is autoscend using all my clovers for levelling and not in Aboo Peak/Smut Orcs/Zeppelin Protesters" that comes our way from now on.

>This was in an avatar path with no access to NC as well, saving many adv.

I suspect optimising for 0 skillpoint Ed runs is going to be less than optimal in 99.99% of all other runs.

Reply:
Fair enough.
I will return that bit of code. And might do a future PR that replaces it with a full calculation function that determines when to it should and should not be used.

Reply:
change made